adapters.go

```diff
@@ -51,26 +78,24 @@ func AdaptPayloadTxsToCosmosTxs(ethTxs []hexutil.Bytes, signTx txSigner, from st
 	if err != nil {
 		return nil, fmt.Errorf("new any with value: %v", err)
 	}
-	depositTx := sdktx.Tx{
+	return &sdktx.Tx{
 		Body: &sdktx.TxBody{
 			Messages: []*codectypes.Any{msgAny},
 		},
-	}
-
-	if signTx != nil {
-		if err := signTx(&depositTx); err != nil {
-			return nil, fmt.Errorf("sign tx: %v", err)
-		}
-	}
+	}, nil
+}
 
+func convertToCosmosTxs(depositTx *sdktx.Tx, nonDepositTxs []hexutil.Bytes) (bfttypes.Txs, error) {
 	depositSDKMsgBytes, err := depositTx.Marshal()
 	if err != nil {
 		return nil, fmt.Errorf("marshal tx: %v", err)
 	}
```

Comment:
I would probably leave this one-off in the "main" function, and change this helper to take only the `nonDepositTxs []hexutil.Bytes`. It feels a little weird for a converter to take two different input types (which is why the adapter functions were already hard to read and think about).